app/models/movie.rb

```diff
@@ -1,3 +1,41 @@
+require 'date'
+
 class Movie < ApplicationRecord
   has_many :parties, dependent: :destroy
+
+  def release_year(release_date)
+    if release_date != ''
```

Comment:
good catch on movies without a release date. We can also refactor this into a helper method.